src/util/Permissions.js

```diff
@@ -91,9 +91,10 @@ class Permissions extends BitField {
  * * `USE_APPLICATION_COMMANDS`
  * * `REQUEST_TO_SPEAK`
  * * `MANAGE_THREADS`
- * * `USE_PUBLIC_THREADS`
- * * `USE_PRIVATE_THREADS`
+ * * `USE_PUBLIC_THREADS` (create threads)
+ * * `USE_PRIVATE_THREADS` (create private threads)
```

Comment:
```suggestion
 * * `USE_PUBLIC_THREADS` (deprecated)
 * * `USE_PRIVATE_THREADS` (deprecated)
```

And add the new ones in the appropriate place.